mailpoet/tests/integration/AdminPages/HelpTest.php

```diff
@@ -0,0 +1,107 @@
+<?php declare(strict_types=1);
+
+namespace integration\AdminPages;
+
+use MailPoet\AdminPages\PageRenderer;
+use MailPoet\AdminPages\Pages\Help;
+use MailPoet\Cron\CronHelper;
+use MailPoet\Entities\NewsletterEntity;
+use MailPoet\Entities\ScheduledTaskEntity;
+use MailPoet\Entities\SendingQueueEntity;
+use MailPoet\Helpscout\Beacon;
+use MailPoet\Newsletter\Sending\ScheduledTasksRepository;
+use MailPoet\Newsletter\Sending\SendingQueuesRepository;
+use MailPoet\Newsletter\Url;
+use MailPoet\Services\Bridge;
+use MailPoet\Tasks\Sending;
+use MailPoet\Test\DataFactories\Newsletter;
+use MailPoet\Test\DataFactories\ScheduledTask as ScheduledTaskFactory;
+use MailPoetVendor\Carbon\Carbon;
+
+class HelpTest extends \MailPoetTest {
+
+  /** @var ScheduledTaskFactory */
+  private $scheduledTaskFactory;
+
+  /*** @var SendingQueuesRepository */
+  private $sendingQueuesRepository;
+
+  /**
+   * @var Help
+   */
+  private $helpPage;
+
+  public function _before() {
+    parent::_before();
+    $this->cleanup();
```

Comment:
Do we need to call cleanup() here considering that we are already calling it on _after()? AFAIK, removing everything from the database after the test is executed is enough and it is not necessary to do the same again before starting the execution of the next test.

Reply:
I've seen tests that leave stuff in DB and make other tests provide falls negatives or positives, and besides cleaning too much is better than cleaning too less!

Reply:
That is a fair point. My concern here, and I haven't measured anything so I'm talking anecdotally, is that by running the delete queries twice for each test we are making our integration test suite unnecessarily slower and I believe we should aim to make it as fast as possible. 

I would suggest that instead of running the cleanup code twice, we strive to find and fix the tests that are not cleaning up after themselves whenever we run into errors caused by such tests. Did you run into errors with your tests because of other tests that were not removing data after they were executed? If so, I'd be happy to help you to trace down which tests are causing the problem. Usually creating a MailPoetTest::_before() method that prints the result of a query that counts the number of existing entries of a given entity and then running the whole test suite is enough to find the tests that are leaving data behind. One caveat is that it is necessary to use `codecept_debug()` instead of the normal PHP functions like `var_dump()`.

If there are no errors when running the tests that you created without calling `cleanup()` inside `_before()`, in my opinion, we should remove it because of the added overhead.